docs/SECURITY-EMERGENCY-PLAN.md

```
@@ -0,0 +1,337 @@
+
+### Hour 24-36: Biometric Authentication
+```typescript
+// 2. Create BiometricAuth.ts
+import * as LocalAuthentication from 'expo-local-authentication';
+
+export class BiometricAuth {
+  static async authenticate(): Promise<boolean> {
+    const hasHardware = await LocalAuthentication.hasHardwareAsync();
+    if (!hasHardware) return false;
+
+    const isEnrolled = await LocalAuthentication.isEnrolledAsync();
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)